components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1.common/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/common/mappings/ImportUtils.java

```diff
@@ -277,6 +285,21 @@ public static API importApi(String extractedFolderPath, APIDTO importedApiDTO, B
         }
     }
 
+    /**
+     * This method sets the operations which were retrieved from the swagger definition to the API DTO.
+     *
+     * @param apiDto             API DTO
+     * @param response          API Validation Response
+     * @throws APIManagementException If an error occurs when retrieving the URI templates
+     */
+    private static void setOperationsToDTO(APIDTO apiDto, APIDefinitionValidationResponse response)
+            throws APIManagementException {
+        List<URITemplate> uriTemplates = new ArrayList<>();
+        uriTemplates.addAll(response.getParser().getURITemplates(response.getJsonContent()));
+        List<APIOperationsDTO> apiOperationsDtos =APIMappingUtil.fromURITemplateListToOprationList(uriTemplates);
```

Comment:
Code formatting issue in '='

Reply:
Fixed via https://github.com/wso2/carbon-apimgt/pull/9480/commits/5348d62a58ef3f675074a88af5b2e2bf607660db